### ISSUE TITLE
[name] logging

### ISSUE THREAD

**Issue by verth** [trivago/parallel-webpack#14]
I have an environment where I build 3 entry points and use '[name]' for `output`, so I set up the parallel with variants. It's working great, build time went down and the config was easily shared. Good stuff! Only downside is that in the console the processes are all logged as '[name] started ...'.

I can't really use an extra variant property (ex. `name`) because I'm only interested in those 3 builds, the others would only provide overhead, or am I missing something crucial here? If not, I think it would be fairly easy to implement some sort of name resolver / fallback, something like:

```javascript
// webpackWorker.js
function getAppName(webpackConfig) {
    var entryName = webpackConfig.name || webpackConfig.output.filename;
    return entryName.indexOf('[name]') === -1 ? entryName : entryName.replace('[name]', Object.keys(webpackConfig.entry));
}
```

**Comment by pago** [comment]
Could you provide a minimal config file so that I have something to test with?

**Comment by verth** [comment]
Yes ofcourse, see a config example below. I also made a [temporary repo](https://github.com/verth/webpack-parallel-config-examples) with the probem, the hotfix, and some minimal configs, in case it might help.

```javascript
'use strict';

const createVariants = require('parallel-webpack').createVariants;
const baseOptions = {
    output: {
        path: './dist',
        filename: '[name].transform.js'
    },
    module: {
      loaders: [
        {
          test: /\.jsx?$/,
          exclude: /(node_modules|bower_components)/,
          loader: 'babel?presets[]=es2015',
        }
      ]
    }
}

const getVariantMultiCompilerConfig = () => {
    const variants = {
        entry: [
            {windows: './src/windows.js'},
            {unix: './src/unix.js'}
        ]
    };

    const createConfig = options => {
        return {
            entry: options.entry,
            output: options.output,
            module: options.module
        };
    }

    return createVariants(baseOptions, variants, createConfig);
}

module.exports = getVariantMultiCompilerConfig();
```

**Comment by pago** [comment]
Thank you! I'll have a look at it during the next week. Awesome work and I'm sure the repository will be extremely helpful. 👍

**Comment by shaharmor** [comment]
Any update on this?

**Comment by pago** [comment]
This feature is included in `parallel-webpack@1.5.0` for configurations where there is only a single entry point.

For other use cases (i.e. if you have more than a single entry point), please consider using the `name` property, i.e.

```js
module.exports = {
    name: 'unix.bundle.js',
    entry: {
        unix: '...'
    }
};
```
